src/radix_connect/mobile/deep_link_parsing/request.rs

```diff
@@ -0,0 +1,162 @@
+use crypto::signatures::ed25519::Signature;
+
+use super::super::session::session_id::SessionID;
+use crate::prelude::*;
+use hex::ToHex;
+
+#[derive(Debug, PartialEq)]
+pub struct RadixConnectMobileRequest {
```

Comment:
missing doc, is this request always sent FROM Dapp to Host, and parsed by Sargon? Maybe put something like that?